src/NServiceBus.AcceptanceTests/ApprovalFiles/When_endpoint_is_warmed_up.Make_sure_things_are_in_DI.approved.txt

```diff
@@ -1,10 +1,10 @@
 ----------- Actively used components (Find ways to stop accessing them)-----------
-NServiceBus.Hosting.HostInformation - SingleInstance
 NServiceBus.StorageInitializer+CallInit - SingleInstance
 NServiceBus.Transport.IDispatchMessages - SingleInstance
 NServiceBus.Unicast.MessageHandlerRegistry - SingleInstance
 ----------- Likely unused components (Remove in next major if possible) -----------
 NServiceBus.CriticalError - SingleInstance
+NServiceBus.Hosting.HostInformation - SingleInstance
```

Comment:
is there an alternative for users?

Reply:
Not at this stage, so if we stop registering in DI we would have to expose it on the FeatureSetupContext